docs/sources/configuration/integrations/snowflake-config.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
+- /docs/agent/latest/configuration/integrations/snowflake-config/
+title: snowflake_config
+---
+
+# snowflake_config
+
+The `snowflake_config` block configures the `snowflake` integration,
+which is an embedded version of
+[`snowflake-prometheus-exporter`](https://github.com/grafana/snowflake-prometheus-exporter). This allows the collection of [Snowflake](https://www.snowflake.com/) metrics.
+
+Full reference of options:
+
+```yaml
+  # Enables the apache_http integration, allowing the Agent to automatically
+  # collect metrics for the specified snowflake account.
```

Comment:
Is it enabling the apache_http integration?  I thought this was just the snowflake integration but maybe it needs that integration, too?

Reply:
Good catch; I had copied to common settings from the apache_http doc and missed this!